### PR TITLE
feat(validator): change validator reward scalar to 1

### PR DESCRIPTION
**Pull request by 0xHansLee**
# Description

Changed temp validator reward ratio to 1.

### CODE REVIEW

File: components/node/rollup/derive/validator_reward.go [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package derive
 
-const TempValidatorRewardScalar = uint64(2000) // Denominator is 10000, so the ratio is 20%
+const TempValidatorRewardScalar = uint64(10000) // Denominator is 10000, so the ratio is 100%
```
Comment (seolaoh) [inline components/node/rollup/derive/validator_reward.go:3]: Please change L7-L8 like below.
```
	// This section requires additional logic to calculate the scalar.
```

File: e2e/system_test.go [hunk 2]
```diff
@@ -1606,7 +1606,7 @@ func TestFees(t *testing.T) {
 
 	// Tally Protocol fund
 	protocolFee := new(big.Int).Sub(fee, reward)
-	require.Equal(t, protocolFee, protocolVaultDiff, "protocol fund mismatch")
+	require.Equal(t, protocolFee.Uint64(), protocolVaultDiff.Uint64(), "protocol fund mismatch")
```
Comment (seolaoh) [inline e2e/system_test.go:1609]: I wonder why changed the type.

**Review by seolaoh — APPROVED** [review]
LGTM

**Review by Pangssu — APPROVED** [review]
LGTM

**Pull request merged**